cmd/gen-release-notes/templates/minorReleaseNotes.md

```diff
@@ -1,7 +1,13 @@
 ---
 title: Change Notes
```

Comment:
Do we need to add the version here as well?

Reply:
Similar in upgradeNotes.md

Reply:
In istio/istio.io#9649 we decided not to include the version in the title, so will skip that here for now.